_data/build-research.py

```diff
@@ -44,22 +43,14 @@
   # paper publisher
   new_paper["publisher"] = old_paper.get("publisher", "")
   # paper date
-  date = old_paper.get("issued").get("date-parts")[0]
-  if len(date) > 0:
-    new_paper["year"] = date[0]
-  if len(date) > 1:
-    new_paper["month"] = date[1]
-    new_paper["month_name"] = calendar.month_abbr[new_paper["month"]]
-  if len(date) > 2:
-    new_paper["day"] = date[2]
+  date_parts = old_paper.get("issued").get("date-parts")[0]
+  date_parts += (3 - len(date_parts)) * [1] # default month and day to 0
+  new_paper["date"] = "-".join([str(part) for part in date_parts])
```

Comment:
```suggestion
  new_paper["date"] = "-".join([str(part) for part in date_parts])
```
Is there a reason not to make this as date through the datetime.date class?

Reply:
I found that `datetime` just added `0` paddings before days and months, and I found that jekyll didn't care if they were there or not.

Reply:
@dongbohu What's your thought on this. I have no experience with this datetimel library.

Reply:
I think a `date` or `datetime` object will be better, but if jekyll doesn't care, then a plain string is acceptable.